app/assets/src/components/utils/colormaps/Colormap.js

```diff
@@ -0,0 +1,18 @@
+export class Colormap {
```

Comment:
could you add comments to these

Reply:
Done... what is the main confusion?
is the comment sufficiently clear?